infra/cifuzz/build_fuzzers_test.py

```diff
@@ -128,6 +128,28 @@ def test_correct_host_repo_path(self, _, __):
     self.assertEqual(os.path.basename(builder.host_repo_path),
                      os.path.basename(image_repo_path))
 
+  @mock.patch('clusterfuzz_deployment.ClusterFuzzLite.upload_latest_build',
+              return_value=True)
+  def test_upload_build_disabled(self, mock_upload_build):
```

Comment:
According to the convention I use (not sure if it's good or just personal) this param should be called `mocked_upload_build`. Should I switch convention to calling them `mock_` instead of `mocked_`?
IIRC the reason why I do that is because if I define a function to use as a side-effect I usually call it `mock_`

Reply:
Let's just use `mock_`. It's shorter and just as clear, and I don't think there's that much of a benefit to distinguish the case you pointed out.